front/migrations/20240626_backfill_content_fragment_content_type.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+UPDATE content_fragments
+WHERE "contentType" = 'file_attachment'
+SET "contentType" = CASE
+	WHEN title LIKE '%.csv' THEN 'text/csv'
+	WHEN title LIKE '%.md' THEN 'text/markdown'
+	WHEN title LIKE '%.pdf' THEN 'application/pdf'
+	WHEN title LIKE '%.PDF' THEN 'application/pdf'
+	ELSE 'text/plain' END;
```

Comment:
Can you shed some lights on what's the else cover?

Reply:
From the look of it, it basically matches files without any extension.
See here: https://metabase.dust.tt/question/623-content-fragment-no-known-extension

Reply:
Ok they very likely are either PDFs or plain text.

Reply:
Yup